lib/py/tests/edge_context_tests.py

```diff
@@ -128,6 +131,20 @@ def test_invalidated_authentication_token(self):
             with self.assertRaises(NoAuthenticationError):
                 getattr(token, attr)
 
+    def test_validated_service_authentication_token(self):
+        payload = {
+            "sub": "service/test-service",
+            "exp": 1574458470,
+            "obo": {"aid": "t2_deadbeef", "roles": ["admin"]},
+            "sea": True,
+        }
+
+        token = ValidatedAuthenticationToken(payload)
+        self.assertEqual(token.subject, "service/test-service")
+        self.assertEqual(token.on_behalf_of_roles, ["admin"])
+        self.assertEqual(token.on_behalf_of_id, "t2_deadbeef")
+        self.assertEqual(token.requests_elevated_access, True)
```

Comment:
🔕  since these tests run with pytest, I would probably recommend using regular assertion statements (e.g. `assert token.subject == "service/test_service"`). pytest will do its magic assertion rewriting to produce a better error output than the TestCase assert mehtods.

Not necessary to change though, I know you're following the pre-existing pattern in this file.

Reply:
makes sense - i can at least update the new changes to follow the new style

Reply:
take it back - looks like the linters were complaining so going to keep it as is - if we want to update the linters in drone, will defer that into a separate change